### PR TITLE
feat(storage.cold-archive): Add manage archive view

### PR DESCRIPTION
**Pull request by brofik**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/MANAGER-9069` 
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-10579 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/cold-archives.constants.js [hunk 1]
```diff
@@ -27,6 +27,27 @@ export const CHECK_PRICES_DOC_LINK = {
...
+    'https://docs.ovh.com/ca/storage/object-storage/s3/getting-started-with-object-storage/',
+  EN:
+    'https://docs.ovh.com/en/storage/object-storage/s3/getting-started-with-object-storage/',
+  IT:
+    'https://docs.ovh.com/it/storage/object-storage/s3/getting-started-with-object-storage/',
+  PL:
+    'https://docs.ovh.com/pl/storage/object-storage/s3/getting-started-with-object-storage/',
+  PT:
+    'https://docs.ovh.com/pt/storage/object-storage/s3/getting-started-with-object-storage/',
+};
```
Comment (y4nnL) [inline packages/manager/modules/pci/src/projects/project/storages/cold-archive/cold-archives.constants.js:49]: Missing IR, GB, QC, US, WS, WW, AU, SG, ASIA,

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/containers.html [hunk 2]
```diff
@@ -21,7 +21,7 @@
             data-sortable
             data-filterable
         >
-        <span ng-bind="::$row.formatDate"></span>
+            <span ng-bind="::$row.formatDate"></span>
```
Comment (y4nnL) [inline packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/containers.html:24]:
```suggestion
            <span data-ng-bind="::$row.formatDate"></span>
```

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/containers.html [hunk 3]
```diff
@@ -153,6 +153,21 @@
                     data-translate="{{:: 'pci_projects_project_storages_cold_archive_containers_add_container_label' }}"
                 ></span>
             </button>
+            <button
+                class="oui-button oui-button_primary"
```
Comment (y4nnL) [inline packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/containers.html:157]: Missing `type="button"` attribute

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/manage-archive/manage-archive.controller.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+export default class ColdArchiveContainersManageArchiveController {
+  /* @ngInject */
+  constructor($translate, coreConfig) {
+    this.$translate = $translate;
+    this.coreConfig = coreConfig;
+  }
+
+  $onInit() {
+    this.trackPage('containers::container::manage');
```
Comment (y4nnL) [inline packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/manage-archive/manage-archive.controller.js:11]: Is it possible to use the `atInternet` property inside the routing so we could get rid of this ?
```javascript
atInternet: {
  rename: 'containers::container::manage',
},
```

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/manage-archive/manage-archive.routing.js [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+          return PciStoragesColdArchiveService.getArchiveEndpoint(
+            projectId,
+            REGION,
+          ).then(({ data }) => {
+            return data?.services[0]?.endpoint;
+          });
+        } catch (err) {
+          return '';
+        }
+      },
```
Comment (y4nnL) [inline packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/manage-archive/manage-archive.routing.js:29]: As your function is not marked with `async` please use the promise version of `$http.get`
Also, the `return` statements can be omitted for readability 
```suggestion
      endpoint: /* @ngInject */ (PciStoragesColdArchiveService, projectId) =>
        PciStoragesColdArchiveService.getArchiveEndpoint(projectId, REGION)
          .then(({ data }) => data?.services[0]?.endpoint)
          .catch(() => ''),
```

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/translations/Messages_fr_FR.json [hunk 6]
```diff
@@ -29,5 +30,12 @@
...
-  "pci_projects_project_storages_cold_archive_container_action_virtual_host": "Virtual Host"
+  "pci_projects_project_storages_cold_archive_container_action_virtual_host": "Virtual Host",
+  "pci_projects_project_storages_cold_archive_container_manage": "Monitorer vos archives depuis l’espace client visuel fourni par OVH Cold Archive",
+  "pci_projects_project_storages_cold_archive_container_manage_documentation_link": "Comment configurer votre CLI",
+  "pci_projects_project_storages_cold_archive_container_manage_description_1": "Vos opérations d’archivage, comme le téléchargement, la restauration ou la suppression de vos données, se font par ligne de commandes.",
+  "pci_projects_project_storages_cold_archive_container_manage_description_2": "Par exemple, pour télécharger des données, telles que des photos, des vidéos et d’autres documents, vous devez utiliser la CLI, le SDK ou l’API S3",
+  "pci_projects_project_storages_cold_archive_containers_container_manage_endpoint_label": "Endpoint",
+  "pci_projects_project_storages_cold_archive_containers_container_manage_region_label": "Region",
```
Comment (y4nnL) [inline packages/manager/modules/pci/src/projects/project/storages/cold-archive/translations/Messages_fr_FR.json:39]:
```suggestion
  "pci_projects_project_storages_cold_archive_containers_container_manage_region_label": "Région",
```

**Review by y4nnL — CHANGES_REQUESTED** [review]

**Comment by y4nnL** [comment]
Looks like `packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/manage-archive/manage-archive.constants.js` is empty. Can we delete it ?

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/containers.html (hunk 2 above)
Comment (brofik) [reply packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/containers.html]: Done

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/containers.html (hunk 3 above)
Comment (brofik) [reply packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/containers.html:157]: Done

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/translations/Messages_fr_FR.json (hunk 6 above)
Comment (brofik) [reply packages/manager/modules/pci/src/projects/project/storages/cold-archive/translations/Messages_fr_FR.json]: Done

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/manage-archive/manage-archive.controller.js (hunk 4 above)
Comment (brofik) [reply packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/manage-archive/manage-archive.controller.js:11]: Done.

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/manage-archive/manage-archive.routing.js (hunk 5 above)
Comment (brofik) [reply packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/manage-archive/manage-archive.routing.js:29]: Done.

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/cold-archive.service.js [hunk 7]
```diff
@@ -169,6 +169,10 @@ export default class PciStoragesColdArchiveService {
       );
   }
 
+  getArchiveEndpoint(serviceName, regionName) {
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/storages/cold-archive/cold-archive.service.js:172]: **nitpick:** function name is not meaningful as it gets the region info.

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/manage-archive/manage-archive.html [hunk 8]
```diff
@@ -0,0 +1,45 @@
+<oui-modal
+    data-heading="{{:: 'pci_projects_project_storages_cold_archive_container_manage' | translate }}"
+    data-primary-action="$ctrl.onManageContainerClick()"
+    data-primary-disabled="$ctrl.isLoading"
+    data-secondary-action="$ctrl.cancel()"
+    data-on-dismiss="$ctrl.cancel()"
+    data-secondary-label="{{:: 'pci_projects_project_storages_cold_archive_containers_container_manage_close_label' | translate }}"
+    data-loading="$ctrl.isLoading"
+>
+    <p>
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/manage-archive/manage-archive.html:10]: **nitpick:** Isn't this redundant? `<p>`

**Review by anooparveti — CHANGES_REQUESTED** [review]
Could you please remove empty file `packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/manage-archive/manage-archive.constants.js`?

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/manage-archive/manage-archive.html (hunk 8 above)
Comment (brofik) [reply packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/manage-archive/manage-archive.html:10]: I used the block tag <p> in order to have a proper spacing for the inline tag <a>.

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/cold-archive.service.js (hunk 7 above)
Comment (brofik) [reply packages/manager/modules/pci/src/projects/project/storages/cold-archive/cold-archive.service.js:172]: I changed it to "getArchiveRegionDetails()".

**Review by anooparveti — APPROVED** [review]

**Pull request merged**